base_layer/wallet/src/storage/sqlite_db/wallet.rs

```diff
@@ -669,7 +669,7 @@ fn check_db_encryption_status(
                         WalletStorageError::InvalidEncryptionCipher
                     })?;
                 } else {
-                    error!(
+                    info!(
```

Comment:
The message looks like it should be an error/warn. Under what conditions does it appear?

Reply:
Changed back to error. When the master key is encrypted and passphrase is not provided the wallet will fail to start. So I think error is appropriate in this case.